Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+         public static string ReadInputString(string message)
+        {
+            Console.WriteLine(message);
+            string value = Console.ReadLine();
+            
+            if (string.IsNullOrEmpty(value))
+            {
+                Console.Write("Name cannot be empty.");
+                value = "-";
+            }
+
+            return value;
+        }
+
+        public static float ReadInputFloat(string message)
+        {
+            Console.WriteLine(message);
+            string input = Console.ReadLine();
+            float value = ValidateFloat(input);
+            if (value < 0) PrintInvalidNumber(input);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4917496)

Reply:
Do I need to change this? Kai said in the lessons that in this case he prefers 
`if (value < 0) PrintInvalidNumber(input);`
insted of 
if (value < 0) 
{
    PrintInvalidNumber(input);
}

Reply:
This is a very hot topic and my advice would be to follow the rule. There was a rocket crashing because of this. Not because of it not working but because of someone not paying very much attention and writing another line under the if clause which didn't end up being in the then part.

So dangerous code never should make use of that.